autouri/abspath.py

```diff
@@ -39,6 +40,21 @@ def __init__(self, uri, thread_id=-1):
     def is_valid(self):
         return os.path.isabs(self._uri)
 
+    def rmdir(self, dry_run=False, num_threads=None, no_lock=False):
```

Comment:
`num_threads` isn't used in the function, would be good to take it out of the signature and use `*args/**kwargs` if it might get passed.

Reply:
Thanks, removed `num_threads` from the args list.